### PR TITLE
tests(Settings_v2): add tests to enable and disable 2fa

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Because
`Added tests for Settings redesign to`
`- Enable the 2FA`
`- Disable the 2FA`

## This pull request

`-Closes task FXA-1565: Add/Revoke 2FA`

## Issue that this pull request solves

Closes: # (issue number)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.
```
While writing this test I found an issue FXA-2765.
At line 116 and 117 I have made a comment about it as well. Once the bug fixes the tests can be fixed as well.
@chenba  please review! Thanks!
```

### CODE REVIEW

**Comment by ashrivastava-qa** [comment]
@chenba have incorporated all the comments you made. 
Only thing that couldn't be done in this PR was splitting the navigateToSettings() into creating user() and navigating() that @vbudhram suggested as that will affect a few other tests as well and I didn't want to add everything here. I will create another PR for that change.

File: packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+'use strict';
+
+const { describe, it, before } = intern.getPlugin('interface.bdd');
+const selectors = require('../lib/selectors');
+const FunctionalHelpers = require('../lib/helpers');
+const FunctionalSettingsHelpers = require('./lib/helpers');
+const { navigateToSettingsV2 } = FunctionalSettingsHelpers;
+const { generateTotpCode } = FunctionalHelpers;
+let secret;
+let securityCode;
```
Comment (chenba) [inline packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js:14]: These two variables do not need to be in this scope.  You can declare and assign them as `const`s where they are first used.

**Review by chenba — APPROVED** [review]
👍  r+ with comment

File: packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js (hunk 1 above)
Comment (ashrivastava-qa) [reply packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js]: Made the change. Thanks for the review! :)

**Pull request merged**